### PR TITLE
basic attribute reporting

### PR DESCRIPTION
**Pull request by tjikkun**

### CODE REVIEW

File: zigate/cluster/core.py [hunk 1]
```diff
@@ -37,6 +38,7 @@ def __init__(self, device, endpoint):
         self._attributes = self.__class__._attributes.copy()
         self.device = device
         self.endpoint = endpoint
+        self.attr_notify = signal('zigate_attr_notify')
```
Comment (blubber) [inline zigate/cluster/core.py:41]: Perhaps we can move the signals to their own module, like `zigate/signals.py` and create one global signal for each event. Then when dispatching the event, add the `Cluster` instance as keyword arg.

And maybe you can give it a more descriptive name like `attribute_changed`.

File: zigate/cluster/on_off.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    return data.decode('utf-8')
+
+def struct_unpack(fmt):
+    st = struct.Struct(fmt)
+    def _(data):
+        return st.unpack(data)[0]
+    return _
+
+
+class OnOff(Cluster):
```
Comment (blubber) [inline zigate/cluster/on_off.py:16]: I wouldn't mind renaming this to `Swtich`,  a slight departure from the Zigate/Zigbee API, but more natural.

**Review by blubber — COMMENTED** [review]
I'm thinking about refactoring the Manager to manage Endpoints instead of Devices. This might make it easier to implement integration with Domoticz and other platforms.

File: zigate/cluster/on_off.py (hunk 2 above)
Comment (tjikkun) [reply zigate/cluster/on_off.py:16]: In that case, what do you want to name cluster 0x0007: On/Off Switch Configuration?
On/Off is less ambiguous than Switch, but if you insist, I am fine with either :)

Comment (blubber) [reply zigate/cluster/on_off.py:16]: mmm ok, guess you're right. Let's leave it at 'OnOff' then.

**Pull request merged**